kitsune/search/v2/search.py

```diff
@@ -159,6 +159,42 @@ def make_result(self, hit):
         }
 
 
+class ProfileSearch(SumoSearch):
+    """Search over User Profiles."""
+
+    def get_index(self):
+        return ProfileDocument.Index.read_alias
+
+    def get_fields(self):
+        return ["username", "name"]
+
+    def get_highlight_fields(self):
+        return []
+
+    def get_filter(self, **kwargs):
+        return DSLQ(
+            "boosting",
+            positive=self.build_query(**kwargs),
+            negative=DSLQ(
+                "bool",
+                must_not=DSLQ("terms", group_ids=kwargs.get("group_ids", [])),
+            ),
+            negative_boost=0.5,
+        )
+
+    def get_highlight_options(self):
+        return {}
+
+    def make_result(self, hit):
+        return {
+            "type": "document",
+            "avatar": getattr(hit, "avatar", None),
+            "username": hit.username,
+            "name": hit.name,
```

Comment:
this needs to be `getattr(hit, "name", None)`, as blank names won't appear in ES

jinja2 then renders `None` as `"None"`, so it might be worth expanding that to `getattr(hit, "name", None) or ""`